src/app/store.ts

```diff
@@ -1,10 +1,12 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The function 'combineSlices' does not exist in '@reduxjs/toolkit'. You should use 'combineReducers' instead to combine the slices.